x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/SecurityBaseRestHandler.java

```diff
@@ -57,8 +56,7 @@ protected final RestChannelConsumer prepareRequest(RestRequest request, NodeClie
     /**
      * Check whether the given request is allowed within the current license state and setup,
      * and return the name of any unlicensed feature.
-     * By default this returns an exception if security is not available by the current license or
-     * security is not enabled.
+     * By default this returns an exception if security is not enabled.
```

Comment:
```suggestion
     * By default this returns an exception if security is disabled.
```

Reply:
happy to change, can you clarify - mainly for my curiosity - why you think "disabled" is preferable to "not enabled" here? It's just javadoc, so there's no fuss, but I'm just wondering if we should change the message of the exception below if "disabled" is clearer